terraform/workspace-variables/review.tfvars

```diff
@@ -10,4 +10,4 @@ paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
 paas_web_app_memory = 512
-paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && rails db:seed && rails s"
+paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && rails db:seed && /app/bin/delayed_job start && rails s"
```

Comment:
Do we need this change?

So far as I see it, we fire off the importer from our seed setup, so I would be surprised if we need to fire it off as a separate command too.

And if we need it for any delayed job to run, hmm, maybe we want it on start of every application, not just review apps.

Reply:
It's the daemon task that runs delayed jobs. In other environments I was wondering if perhaps we wanted it as a different worker instance. Was leaving that for future PR

Reply:
Sure, can be a future PR, so long as we have a trello card for it so it's not forgotten :)